packages/contracts-bedrock/src/universal/CrossDomainMessenger.sol

```diff
@@ -3,6 +3,7 @@ pragma solidity 0.8.15;
 
 // Libraries
 import { Initializable } from "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import { Math } from "@openzeppelin/contracts/utils/math/Math.sol";
```

Comment:
Should we just add the max function locally to avoid needing this dep here?
```solidity
function max(uint256 a, uint256 b) internal pure returns (uint256) {
    return a >= b ? a : b;
}
```

Reply:
Eh, I guess. It seemed easier to include the dep since we have it elsewhere already and it's well tested.